helm-charts/stable/cloudwatch-agent/templates/cloudwatch-agent-daemonset.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# deploy cwagent as daemonset
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "cloudwatch-agent.fullname" . }}
```

Comment:
Add a section for labels here like the following example I commented above: https://github.com/aws/eks-distro-build-tooling/blob/main/helm-charts/stable/amazon-eks-pod-identity-webhook/templates/Service.yaml#L20